content/zh/docs/tasks/configure-pod-container/configure-projected-volume-storage.md

```diff
@@ -0,0 +1,142 @@
+---
+reviewers:
+- jpeeler
+- pmorie
+title: 配置 Pod 使用投射卷存储
+content_template: templates/task
+weight: 70
+---
+
+<!--
+---
+reviewers:
+- jpeeler
+- pmorie
+title: Configure a Pod to Use a Projected Volume for Storage
+content_template: templates/task
+weight: 70
+---
+-->
+
+{{% capture overview %}}
+<!--
+This page shows how to use a [`projected`](/docs/concepts/storage/volumes/#projected) volume to mount
+several existing volume sources into the same directory. Currently, `secret`, `configMap`, `downwardAPI`,
+and `serviceAccountToken` volumes can be projected.
+-->
+
+本文介绍怎样通过[`投射（projected）`](/docs/concepts/storage/volumes/#projected) 卷将现有的多个卷资源挂载到相同的目录。
+当前，`secret`、`configMap`、`downwardAPI` 和 `serviceAccountToken` 卷可以被投射。
+
+{{< note >}}
+<!--
+`serviceAccountToken` is not a volume type.
+-->
+`serviceAccountToken` 不是一种卷类型
+{{< /note >}}
+{{% /capture %}}
+
+{{% capture prerequisites %}}
+{{< include "task-tutorial-prereqs.md" >}} {{< version-check >}}
+{{% /capture %}}
+
+{{% capture steps %}}
+
+<!--
+## Configure a projected volume for a pod
+
+In this exercise, you create username and password Secrets from local files. You then create a Pod that runs one Container, using a [`projected`](/docs/concepts/storage/volumes/#projected) Volume to mount the Secrets into the same shared directory.
+
+Here is the configuration file for the Pod:
+-->
+
+## 为 Pod 配置投射卷
+
+本练习中，您将从本地文件来创建包含有用户名和密码的 Secret。然后创建运行一个容器的 Pod，该 Pod 使用[`投射（projected）`](/docs/concepts/storage/volumes/#projected) 卷将 Secret 挂载到相同的共享目录。
+
+下面是 Pod 的配置文件：
+
+{{< codenew file="pods/storage/projected.yaml" >}}
+
+<!--
+1. Create the Secrets:
+```shell
+    # Create files containing the username and password:
+       echo -n "admin" > ./username.txt
+       echo -n "1f2d1e2e67df" > ./password.txt
+
+    # Package these files into secrets:
+       kubectl create secret generic user --from-file=./username.txt
+       kubectl create secret generic pass --from-file=./password.txt
+```
+-->
+
+1. 创建 Secrets:
+```shell
```

Comment:
should we indent line 75 and similar lines below to make sure the ordered list on line 62 is not broken?

Reply:
I tried just now and found not solved the ordered list problem.

Reply:
https://5c26e3c33bfbcc000b754519--k8s-v1-12.netlify.com/zh/docs/tasks/configure-pod-container/configure-projected-volume-storage/